scripts/model_runner_integ.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+set -e  # Exit immediately if a command exits with a non-zero status
+set -o pipefail  # Exit if any part of a pipeline fails
+
+print_banner() {
+    echo "=========================================="
+    echo "  Running Model Runner Integration Tests  "
+    echo "=========================================="
+}
+
+print_test_passed() {
+    echo "=========================================="
+    echo "       Integration Tests Completed        "
+    echo "=========================================="
+    echo "            All tests passed!             "
+    echo "=========================================="
+}
+
+print_test_failed() {
+    echo "=========================================="
+    echo "        Integration Tests Failed          "
+    echo "=========================================="
+    echo "        Some tests did not pass!          "
+    echo "=========================================="
+}
+
+# Function to run a test and check its result
+run_test() {
+    local description=$1
+    local image=$2
+    local model=$3
+
+    echo "$description"
+    python3 lib/osml-model-runner-test/bin/process_image.py --image "$image" --model "$model" > /dev/null 2>&1
```

Comment:
Why are we not capturing any outputs from this script? This would be beneficial to review the logs straight from Github Workflow if any of the tests failed.

Reply:
Fixed to log outputs on failed jobs